app/controllers/internal/users_controller.rb

```diff
@@ -58,7 +58,7 @@ def full_delete
     @user = User.find(params[:id])
     begin
       Moderator::DeleteUser.call(admin: current_user, user: @user, user_params: user_params)
-      flash[:success] = "@" + @user.username + " (email: " + @user.email + ", user_id: " + @user.id.to_s + ") has been fully deleted. If requested, old content may have been ghostified. If this is a GDPR delete, delete them from Mailchimp & Google Analytics."
+      flash[:success] = "@#{@user.username} (email: #{@user.email.presence || 'no email'}, user_id: #{@user.id}) has been fully deleted. If requested, old content may have been ghostified. If this is a GDPR delete, delete them from Mailchimp & Google Analytics."
```

Comment:
@rhymes Added `.presence` here so we also display "no email" in cases where a user's email is an empty string.